prepare_schemas_deployment/prepare_schemas_deployment.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+#  Copyright (c) 2022 Ecco Sneaks & Data
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+set -Eeuo pipefail
+
+mkdir -p "./$DEPLOYMENT_ROOT/$PROJECT_NAME-schemas/$PROJECT_VERSION/"
+mv -v ./schemas/* "./$DEPLOYMENT_ROOT/$PROJECT_NAME-schemas/$PROJECT_VERSION/"
```

Comment:
looking at this i tend to think we should simply combine this with azcopy command no?